### ISSUE TITLE
Director sources

### ISSUE THREAD

**Issue by jlivingood** [josephlhall/draft-hall-iasa2-struct#40]
JOE: What about a compromise that allows targeted expertise but doesn't
require it... how about an LLC Board of 5, with 0-2 board-appointed
slots that require a unanimous vote of the full board and IESG/IAB
confirmation? They could be non-voting or have other constraints on
their term.

**Comment by jlivingood** [comment]
Jason, I think we need to add Russ' "5 appointed by noncom, no
self-appointed" model to the mix.

**Comment by jlivingood** [comment]
Leslie: For “Finding qualified people”, I do think we need to be able to cast the net more broadly. I think there may be many ways to do it, but having the board identify at least some candidates seems like one reasonable one.

Obviously, having the board identify potential candidates doesn’t mean they have to be the selecting body.

But, a variation on something Joe proposed earlier would be: let the board identify up to 2 candidates themselves, for their initial terms. If those board members want to pursue any further terms, that would have to be done through IAB/IESG/NomCom or whatever other general appointment selection process seems reasonable.

**Comment by jlivingood** [comment]
Leslie: Leslie Daigle <ldaigle@thinkingcat.com> wrote:
    > But, a variation on something Joe proposed earlier would be: let the
    > board identify up to 2 candidates themselves, for their initial
    > terms. If those board members want to pursue any further terms, that
    > would have to be done through IAB/IESG/NomCom or whatever other general
    > appointment selection process seems reasonable.

So basically, a 1-term limit on LLC-board appointed members.
It's not my preferred way, but I can live with that.

**Comment by jlivingood** [comment]
JL: wrote: I still don't see anything that can't be handled by having the
board describe the skills needed each time a position needs filling,
and then allowing the nomcom process those.
[JL] On this point, we could always consider specifying something
like "The LLC Board shall at their option, prior to any IESG, NomCom,
and/or ISOC appointment process, develop recommendations on the
skills, experience and/or people that the Board feels are necessary
to the effective execution of their responsibilities. Such feedback
shall be shared with each respective selection body, and Directors
will make themselves available for meetings or other consultations
with those selection bodies as needed."

**Comment by jlivingood** [comment]
Brian:
For “Finding qualified people”, I do think we need to be able to
cast the net more broadly.  I think there may be many ways to do it, but
having the board identify at least some candidates seems like one
reasonable one.
Obviously, having the board identify potential candidates doesn’t mean
they have to be the selecting body.
But, a variation on something Joe proposed earlier would be:  let the
board identify up to 2 candidates themselves, for their _initial_ terms.
   If those board members want to pursue any further terms, that would
have to be done through IAB/IESG/NomCom or whatever other general
appointment selection process seems reasonable.

Yes, I quite like that. If renewal of co-opted members is only possible
as part of the "normal" process, it becomes possible for the board
to bring in fresh expertise but gives community control for the
longer term.

    Brian

**Comment by jlivingood** [comment]
Brian:

**Comment by jlivingood** [comment]
Me:

All options include these two people on the seven person board:
* 1 IETF Chair or delegate selected by the IESG
* 1 Appointed by the ISOC Board of Trustees

Option 1:
* 3 Appointed by the IETF NomCom, confirmed by the IESG
* 2 Appointed by the LLC board itself, confirmed by IAB, IESG, or NomCom (TBD)

Option 2:
* 4 Appointed by the IETF NomCom, confirmed by the IESG
* 1 Appointed by the LLC board itself, confirmed by IAB, IESG, or NomCom (TBD)

Option 3: (as noted above - I don't think we have consensus support for this one but listing it nonetheless as I imagine I'd be asked why I excluded it)
* 5 Appointed by the IETF NomCom, confirmed by the IESG

**Comment by jlivingood** [comment]
Eric

My ranking is 1 > 2 >> 3.

I feel like most everything that can be said here has already been said, but just to recap: there is ample precedent in other boards for doing this and I've seen it work out well. I don't think there's any significant risk of capture, and I do think that it's actually pretty important to have outside *voting* voices who are able to bring different perspectives to the table.

-Ekr

**Comment by jlivingood** [comment]
Joe:
I agree that some ability to self-appoint is likely a very good idea, based on my own experience. So I'll +1 EKR's ranking here.

(technically, I'm more 1 ~ 2 >> 3)

**Comment by jlivingood** [comment]
Lou & Stephen for 3:
+1


----------
On May 30, 2018 11:13:06 AM Stephen Farrell <stephen.farrell@cs.tcd.ie> wrote:

Hiya,

On 30/05/18 15:22, Livingood, Jason wrote:
Option 3: (as noted above - I don't think we have consensus support
for this one but listing it nonetheless as I imagine I'd be asked why
I excluded it) > * 5 Appointed by the IETF NomCom, confirmed by the IESG
I support option 3 fwiw.

S.
